File: src/main/java/io/github/starwishsama/namelessbot/objects/ShopItem.java
package io.github.starwishsama.namelessbot.objects;

import lombok.Data;

@Data
public class ShopItem {
    private String itemName;
    private double point;
    private String itemCommand;
    private int buyTime;

    public ShopItem() {
    }

    public ShopItem(String itemName, double point, int times, String itemCommand) {
        this.itemName = itemName;
        this.point = point;
        buyTime = times;
        this.itemCommand = itemCommand;
    }

    public ShopItem(String itemName, double point, String itemCommand) {
        this.itemName = itemName;
        this.point = point;
        this.itemCommand = itemCommand;
    }
}

File: src/main/kotlin/io/github/starwishsama/comet/objects/ShopItem.kt
package io.github.starwishsama.comet.objects

class ShopItem {
    var itemName: String
    var botPoint: Int
    var itemCommand: String? = null
    var buyTime: Int
    var money = 0

    constructor(name: String, point: Int, times: Int, itemCommand: String?) {
        itemName = name
        botPoint = point
        buyTime = times
        this.itemCommand = itemCommand
    }

    constructor(name: String, point: Int, times: Int, money: Int) {
        itemName = name
        botPoint = point
        buyTime = times
        this.money = money
    }
}